image_transport/include/image_transport/simple_subscriber_plugin.hpp

```diff
@@ -108,20 +108,31 @@ class SimpleSubscriberPlugin : public SubscriberPlugin
     return base_topic + "/" + getTransportName();
   }
 
+  [[deprecated("Use subscribeImpl(rclcpp::node_interfaces...) instead")]]
```

Comment:
Deprecation Message update
```suggestion
  [[deprecated("Use subscribeImpl(..., rclcpp::QoS, ...) instead")]]
```